tools/codegen/gen_lazy_tensor.py

```diff
@@ -180,26 +181,6 @@ def gen_key(func: FunctionSchema) -> Tuple[str, str]:
         )),
     })
     # Generate headers for shape/dtype funcs for non-meta kernels
-    fm.write_with_template(f'{backend_key}ShapeInference.h', 'ShapeInference.h', lambda: {
-        'lazy_ir_sysinc': [f'#include <{path}>' for path in [
-            "ATen/Tensor.h",
-            "c10/core/ScalarType.h",
-            "c10/util/Optional.h",
-            "torch/csrc/lazy/core/ir.h",
-            "torch/csrc/lazy/core/shape.h",
-            "vector",
-        ]],
-        'lazy_ir_inc': [],
-        'DispatchKey': backend_key,
-        'dispatch_namespace': backend_key.lower(),
-        'func_declarations': list(concat_map_codegen(
-            dest.GenLazyShapeInferenceDefinition(backend_indices[backend_key],
-                                                 tensor_class),
-            grouped_native_functions,
-            codegenInplaceVariant=True
```

Comment:
but it's not quite a duplicate: the remaining copy does not have codegenInplaceVariant=True - are you sure which version is correct to keep?

Reply:
The one on the branch does't have the `codegenInplaceVariant=True` which I assume is the latest one: https://github.com/pytorch/pytorch/blob/lazy_tensor_staging/tools/codegen/gen_lazy_tensor.py.

Reply:
ok. i double checked, i think this is right.